image_generation/stable_diffusion_1_5/cpp/src/main.cpp

```diff
@@ -397,10 +396,11 @@ int32_t main(int32_t argc, char* argv[]) try {
 
     for (uint32_t n = 0; n < num_images; n++) {
         std::uint32_t seed = num_images == 1 ? user_seed : user_seed + n;
-        ov::Tensor noise = randn_tensor(height, width, read_np_latent, seed);
 
         // latents are multiplied by 'init_noise_sigma'
-        ov::Shape latent_shape = noise.get_shape(), latent_model_input_shape = latent_shape;
+        ov::Shape latent_shape = ov::Shape({batch_size, static_cast<size_t>(sample_shape[1].get_length()), height / VAE_SCALE_FACTOR, width / VAE_SCALE_FACTOR});
```

Comment:
nitpick : should we extract `sample_shape[1].get_length()` as variable `unet_in_channels` to denote relationship between dimensions? 
Otherwise, people don't know why this dimension is taken..